### PR TITLE
Portals do not go on supply shuttles

### LINKED ISSUE #29494: You can get on the cargo shuttle while its in transit with this one simple trick!
step 1: Get a bluespace wormhole projector
step 2: shoot one wormhole on the shuttle
step 3: launch shuttle
step 4: go through portal onto shuttle
step 5: ???
step 6: profit

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #29494

### CODE REVIEW

**Comment by BeeSting12** [comment]
This can be worked around by shooting the portal beamgun before it launches in the five second window you have. The projectile will be on the shuttle, but not a portal yet. The shuttle will launch, the portal will go down, and then you can do it as described in the issue report.

File: code/modules/shuttle/supply.dm [hunk 1]
```diff
@@ -18,7 +18,9 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		/obj/structure/destructible/clockwork/powered/clockwork_obelisk,
 		/obj/item/device/warp_cube,
 		/obj/machinery/r_n_d/protolathe, //print tracking beacons, send shuttle
-		/obj/machinery/autolathe //same
+		/obj/machinery/autolathe, //same
+		/obj/item/projectile/beam/wormhole,
```
Comment (ChangelingRain) [inline code/modules/shuttle/supply.dm]: good try but I'm not an idiot

Comment (BeeSting12) [inline code/modules/shuttle/supply.dm]: noticed that once i posted it :P

**Comment by Jalleo** [comment]
If there is a way to harvest a soil or tray once it starts moving that should be prevented too.

**Comment by Tacolizard** [comment]
portal does not into supply shuttle

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**